libtesting-ui/src/main/java/com/mapbox/navigation/testing/ui/DualLocationMocker.kt

```diff
@@ -0,0 +1,38 @@
+package com.mapbox.navigation.testing.ui
+
+import android.location.Location
+
+/**
+ * Mocker that pushes updates to both location provider implementations
+ * so that the sample is available regardless where the running location engine tries to get it from.
+ *
+ * Introduction of this dual mocker was needed to initially work around CORESDK-1528
+ * so that the last location is available in both Google fused and Android location providers at the same time.
+ */
+internal class DualLocationMocker(
```

Comment:
NIT: I'd introduce a `CompositeLocationMocker` instead with a list of providers instead of hardcoding 2 of them.

Reply:
I don't think we need that atm, this was just a quick change to unblock the testing. If we need to expand in the future, we can do that.